app/src/main-process/app-window.ts

```diff
@@ -20,10 +20,13 @@ export class AppWindow {
   private minWidth = 960
   private minHeight = 660
 
+  private shouldMaximizeOnShow: boolean = false
```

Comment:
We don't typically annotate types that are easily inferable like this one. Let's also add a little pointer for our future selves.

```suggestion
  // See https://github.com/desktop/desktop/pull/11162
  private shouldMaximizeOnShow = false
```